### ISSUE TITLE
'funannotate.library' has no attribute 'log

### ISSUE THREAD

**Issue by abretaud** [nextgenusfs/funannotate#621]
**Are you using the latest release?**
Yes

**Describe the bug**
I entered an empty species name (by mistake), and got this error:

```
Traceback (most recent call last):
  File "/home/abretaud/miniconda3/envs/__funannotate@1.8.7/lib/python3.7/site-packages/funannotate/aux_scripts/tbl2asn_parallel.py", line 206, in <module>
    args.isolate, args.strain, args.tbl2asn, args.version, args.cpus)
  File "/home/abretaud/miniconda3/envs/__funannotate@1.8.7/lib/python3.7/site-packages/funannotate/aux_scripts/tbl2asn_parallel.py", line 106, in runtbl2asn_parallel
    lib.log.error("tbl2asn error: organism not specified")
AttributeError: module 'funannotate.library' has no attribute 'log'
```
No big deal as the error is understandable, but just in case you'd want to fix the error reporting

(by the way, I found this while writing a Galaxy tool for funannotate if you're interested)

**Comment by nextgenusfs** [comment]
Cool.  I'm not a galaxy user so don't know what happens behind the scenes, but given the complexity of dependencies here I suspect a wrapper will be somewhat difficult (apologies in advance :)). 

There was one outstanding issue I wanted to fix before tagging the next release, which will have your fixes here. But in the meantime let me know if there are others and I'll fix.

**Comment by abretaud** [comment]
Galaxy uses Conda packages, so managing the dependencies is not so hard, except for GeneMark and its license...
Cool if there's a new release soon! I'll continue making tests this week, I'll let you know if I find other problems

**Comment by abretaud** [comment]
So, the Galaxy tool is in this PR now: https://github.com/galaxyproject/tools-iuc/pull/3846

**Comment by nextgenusfs** [comment]
fixed in v1.8.9 -- however, having problems with bioconda build.

**Comment by nextgenusfs** [comment]
Thanks again @abretaud for help with the bioconda build -- hopefully its working.
